SPECS/SymCrypt-OpenSSL/SymCrypt-OpenSSL.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        The SymCrypt engine for OpenSSL (SCOSSL) allows the use of OpenSSL with SymCrypt as the provider for core cryptographic operations
 Name:           SymCrypt-OpenSSL
 Version:        1.5.0
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
Harmless, but is the bump necessary? We're only adding a debuginfo package, so nothing really changed for SymCrypt-OpenSSL.

Reply:
This bump is part of this comment - https://github.com/microsoft/azurelinux/pull/10075#discussion_r1712170768

Reply:
Looks like I misunderstood the comment. Will unbump these.